dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/TransactionHelper.java

```diff
@@ -0,0 +1,128 @@
+package io.dockstore.webservice.helpers;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * See the end of Section 13.2.1
+ * https://docs.jboss.org/hibernate/core/4.3/manual/en-US/html/ch13.html
```

Comment:
This looks pretty old, copyright 2015. 
Current version is 5.6 
Was able to find a version of the docs that lets you select a version of hibernate at https://docs.jboss.org/hibernate/orm/5.6/userguide/html_single/Hibernate_User_Guide.html#transactions 
Looks like they've reorganised some sections though, hopefully the newer version makes things easier?